### DISCUSSION TITLE
Video DownloadHelper Addon and jlesage Firefox Docker

### DISCUSSION THREAD

**Discussion by Verwundert** [jlesage/docker-firefox#256] [Q&A]
I wanted to use Video Download Helper Addon (https://addons.mozilla.org/en-US/firefox/addon/video-downloadhelper/)  and found out that I need to install a part via SSH in Linux. 
The installation doesn’t work in Docker CLI, and it says ‘Command not found.’ Does anyone have any ideas?
apk add curl
apk add bash
apk add file

Firefox3:/tmp# curl -sSLf https://github.com/aclap-dev/vdhcoapp/releases/latest/download/install.sh | bash
Downloading: https://github.com/aclap-dev/vdhcoapp/releases/latest/download/vdhcoapp-linux-x86_64.tar.bz2
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 52.8M  100 52.8M    0     0  8307k      0  0:00:06  0:00:06 --:--:-- 9085k
Extracting tarball…
/tmp/vdhcoapp-GjdHjh: bzip2 compressed data, block size = 900k
Registering CoApp
bash: line 76: /root/.local/share/vdhcoapp/vdhcoapp: cannot execute: required file not found
Firefox3:/tmp# 

![Bildschirmfoto 2024-09-09 um 18 32 44](https://github.com/user-attachments/assets/84827788-4bcf-4c96-8f4d-8f65cb0df9b4)

**Comment by strunzoz** [comment]
what files are inside vdhcoapp-linux-x86_64.tar.bz2 ?
